src/components/MarkdownPage/MarkdownPage.js

```diff
@@ -19,8 +19,20 @@ import toCommaSeparatedList from 'utils/toCommaSeparatedList';
 import {sharedStyles} from 'theme';
 import createOgUrl from 'utils/createOgUrl';
 
+type Props = {
+  authors: Array<string>,
+  createLink: Function,
+  date?: string,
+  enableScrollSync?: boolean,
+  ogDescription: string,
+  location: Object,
+  markdownRemark: Object,
+  sectionList: Array<Object>,
```

Comment:
Would be good to add a TODO here to better type this once we Flow-type the `Section` component

Reply:
Sorry, I've been out of touch for the past couple of days. Yes, clearer..I think I was confused because it was highlighting three lines (and they were mostly the same three lines)...Meh, anyway. Pushed. Thanks for that Location type. I didn't know flow had one already. Sweet.